src/modules/media/elasticsearch/consts.ts

```diff
@@ -23,32 +24,35 @@ export const READABLE_TO_ELASTIC_FILTER_NAMES: { [prop in keyof SearchFilters]:
 
 export enum QueryType {
 	TERM = 'term',
+	TERMS = 'terms',
 	RANGE = 'range',
 	MATCH = 'match',
```

Comment:
Are these interpretations correct? If not: add correct comments
```suggestion
	TERM = 'term', // Search for a single term exactly
	TERMS = 'terms', // Must match at least one term exactly
	RANGE = 'range', // Date range or duration range
	MATCH = 'match', // Text based fuzzy search
```